### PR TITLE
Fixed spacing and errors on the song list.

### PR DESCRIPTION
**Pull request by JackpotSHUnited**
New to github so there was some trial and/or error involved, but the song list should be perfectly fixed and up-to-date.

### CODE REVIEW

**Comment by bb010g** [comment]
Can you [squash the commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)?

**Comment by JackpotSHUnited** [comment]
Does it have to be me that does it? I'm looking at that link and I just started drooling in stupidity.

**Comment by JackpotSHUnited** [comment]
rebase

**Comment by bb010g** [comment]
I can walk you through it on the Gitter.

**Comment by somealternateaccount** [comment]
When you're squashing these commits please add a description to the final commit regarding what changes you made so the people of the future don't have to dig through a diff regarding what changes you made.

**Comment by ghost** [comment]
Is this being worked on? Perhaps it would be easiest if Jackpot handed off the files for someone else to deal with?

**Comment by JackpotSHUnited** [comment]
There are no files, it's just the TXT file that lists all the song changes. The spacing is currently out of whack, so I fixed up the spacing and corrected a few errors. 

Mushroom Kingdom II (Melee) should say Peach's Castle (Super Smash Bros.)

A second copy of Metal Cavern replaces Gritzy Desert (A10) for some reason.

Those are the only major errors, the rest of it was just consistency fixes (like (Super Smash Bros. Melee) vs. (Melee) for example).

**Comment by somealternateaccount** [comment]
@JackpotSHUnited It would be nice if you could squash the commits before we merge this, and I believe that's the only thing blocking this right now.

edit: We probably also don't want the "Thank you for playing Project M" message. It's a nice touch for people digging around in the files, but I'm not sure if we want to imply that this is a Project M release, because it is not.

**Comment by JackpotSHUnited** [comment]
I seriously have no idea how to use the "rebase" command or whatever it is I'm supposed to do.

**Comment by bb010g** [comment]
What is the article not covering?

File: projectm/pf/sound/strm/SONGS REPLACED.txt [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
-W15 - Super Mario Bros. 3 (Melee)	-> 	Yoshi's Story (Melee)
-X02 - Menu 1    ->   Menu (Project M)
-X08 - Result Screen			->  Chillin' With the Bros. (Results Display)
+A01 - Ground Theme (Super Mario Bros.)        ->  Metal Battle (Melee)
+A02 - Underground Theme (Super Mario Bros.)   ->  Metal Cavern (by MG3)
+A10 - Gritzy Desert                           ->  Metal Cavern (by MG3)
+A13 - Delfino Plaza                           ->  Defino's Secret
+A16 - Ground Theme 2 (Super Mario Bros.)      ->  Metal Mario (N64)
+A17 - Mario Bros.                             ->  Fountain of Dreams (Melee)
+A20 - Mario Circuit                           ->  Bowser's Castle ver. M
```
Comment (bb010g) [inline projectm/pf/sound/strm/SONGS REPLACED.txt:9]: Inconsistent capitalization (see also L15).

**Comment by hhsnopek** [comment]
@JackpotSHUnited 
cmd: `git rebase -i HEAD~3`
change `pick` on the second and third commit to `fixup` or `f`, save and then `git log` (make sure you have the single commit), `git push origin patch-1 -f`

**Comment by ewmb7701** [comment]
Approved. It's just a text file change, so I can easily see everything that got changed.

File: projectm/pf/sound/strm/SONGS REPLACED.txt (hunk 1 above)
Comment (jojmaht) [inline projectm/pf/sound/strm/SONGS REPLACED.txt:9]: I won't validate the changes until this is fixed. I know it's a minute detail but still.

**Comment by bb010g** [comment]
Moved over to #287.

**Pull request closed without merging**